### PR TITLE
add CallAPI test case

### PR DESCRIPTION
**Pull request by DigiCashfasalabad**
please, review the Call API method with JSON response

### CODE REVIEW

File: src/main/java/com/MailinatorTest/MailinatorTest/CallAPI.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.MailinatorTest.MailinatorTest;
+
+public class CallAPI {
+
```
Comment (AltaPredovicAlta-master) [inline src/main/java/com/MailinatorTest/MailinatorTest/CallAPI.java:4]: hi,

your code looks goods magar kuch lek tu

Comment (DigiCashfasalabad) [reply src/main/java/com/MailinatorTest/MailinatorTest/CallAPI.java:4]: g sir

**Comment by DigiCashfasalabad** [comment]
hi,

new API called method added.

**Comment by DigiCashfasalabad** [comment]
hi,

ok its good marh this code

**Pull request merged**